providers/fips/self_test_data.inc

```diff
@@ -955,6 +967,25 @@ static const ST_KAT_PARAM rsa_key[] = {
     ST_KAT_PARAM_END()
 };
 
+static const ST_KAT_PARAM rsa_pub_key[] = {
+    ST_KAT_PARAM_BIGNUM(OSSL_PKEY_PARAM_RSA_N, rsa_n),
+    ST_KAT_PARAM_BIGNUM(OSSL_PKEY_PARAM_RSA_E, rsa_e),
+    ST_KAT_PARAM_END()
+};
+
+static const ST_KAT_PARAM rsa_priv_key[] = {
+    ST_KAT_PARAM_BIGNUM(OSSL_PKEY_PARAM_RSA_N, rsa_n),
```

Comment:
No CRT parameters?

Reply:
They are not used by a primitive test (as far as I could see) - this case is with no padding,, So it is just m^e mod n

Reply:
Really? I believe in the end for the private key decryption the rsa_ossl_private_decrypt() should be called and that will do the CRT computation if the CRT parameters are present.

So there probably should be actually two tests done for the private key operation - one with CRT parameters and one without.

Reply:
You are right I was looking at the public one..

Reply:
Changed test to use a crt key also for private decryption. Removed the extra encrypt/decrypt data. I think it makes more sense testing the same data that was used for encryption for the decryption test.